src/libcore/default.rs

```diff
@@ -137,6 +137,7 @@ pub trait Default {
 
 macro_rules! default_impl {
     ($t:ty, $v:expr) => {
+        /// Creates a new `$t` with value `$v`
```

Comment:
I don't think so now that I think about it. I just removed it in the latest force push. I'm not going to worry about it for now